invenio_resources/service/search/serializers.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-Resources is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio Resources module to create REST APIs."""
+
+from dateutil import parser
+
+
+def es_to_record(record_hit, record_cls):
+    """Transforms an Elasticsearch record hit into a Record."""
+    _id = record_hit["_id"]
```

Comment:
Let's do a deep copy of `record_hit` first here. Otherwise we are mutating `record_hit` which will lead to strange behaviour.